api/models.py

```diff
@@ -35,6 +35,8 @@ class BaseUser(DeferredReflection, Base):
     LastName = Column(String(255))
     AccessRightsSdo = Column(String(255))
     AccessRightsVendor = Column(String(255))
+    Motivation = Column(String(255))
```

Comment:
I would use motivation only for users waiting for approval -> users_temp.
After user validation, access rights are set and it is no longer needed to remember motivation.